Pipfile

```diff
@@ -9,6 +9,7 @@ flake8 = "*"
 [packages]
 django = "*"
 django-crispy-forms = "*"
+pillow = "*"
```

Comment:
where do you use this package?

Reply:
I am using the package Inside models.py, in the 'ImageField'